web/app/js/components/Sidebar.jsx

```diff
@@ -79,18 +78,23 @@ class Sidebar extends React.Component {
     let versionUrl = `https://versioncheck.linkerd.io/version.json?version=${this.props.releaseVersion}?uuid=${this.props.uuid}`;
     this.api.setCurrentRequests([
       ApiHelpers("").fetch(versionUrl),
+      this.api.fetchMetrics(this.api.urlsForResource("all")),
       this.api.fetchMetrics(this.api.urlsForResource("namespace"))
     ]);
 
     // expose serverPromise for testing
     this.serverPromise = Promise.all(this.api.getCurrentPromises())
-      .then(([versionRsp, nsRsp]) => {
+      .then(([versionRsp, allRsp, nsRsp]) => {
+        let statTables = _.get(allRsp, ["ok", "statTables"], []);
+        let rows = _.flatMap(statTables, tb => tb.podGroup.rows);
+        let resourceGroupings = _.groupBy(rows, r => r.resource.type);
         let nsStats = _.get(nsRsp, ["ok", "statTables", 0, "podGroup", "rows"], []);
         let namespaces = _(nsStats).map(r => r.resource.name).sortBy().value();
```

Comment:
I realize this is existing code, but we can also get this by using `processSingleResourceRollup`:
```
        let nsStats = processSingleResourceRollup(nsRsp);
        let namespaces = _(nsStats).map('name').sortBy().value();
```